### PR TITLE
change builder tooling to support emoji sequences

### PR DESCRIPTION
**Pull request by dougfelt**
rough around the edges.  this has some hard-coded paths at the moment that need to be looked at.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -78,7 +78,8 @@ flag-symlinks: $(WAVED_FLAGS)
 
 $(PNG128_FLAGS): flag-symlinks
 
-EMOJI_PNG128 = ./png/128/emoji_u
+#EMOJI_PNG128 = ./png/128/emoji_u
+EMOJI_PNG128 = /tmp/placeholder_emoji_plus/emoji_u
```
Comment (roozbehp) [inline Makefile]: This and the previous line look suspect. Is this intentional?

File: Makefile [hunk 2]
```diff
@@ -91,6 +92,8 @@ endif
 %.ttx: %.ttx.tmpl $(ADD_GLYPHS) $(UNI) $(PNG128_FLAGS)
 	python $(ADD_GLYPHS) "$<" "$@" "$(EMOJI_PNG128)"
 
+
+
```
Comment (roozbehp) [inline Makefile]: Why these two empty lines?

File: build_emoji_set.py [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+import os
+from os import path
+import re
+import shutil
+
+DST = '/tmp/placeholder_emoji_plus'
+
+SRC_PLACEHOLDER = '/usr/local/google/users/dougfelt/emoji_images/placeholder'
+SRC_NOTO = '/usr/local/google/users/dougfelt/newnoto/noto-emoji/png/128'
+SRC_DRAFT = '/usr/local/google/users/dougfelt/emoji_images/png_thurs'
```
Comment (roozbehp) [inline build_emoji_set.py]: These seem to refer to your own home directory.

File: build_emoji_set.py [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+# then copy the draft images to dst, skipping names with parens and
+# after fixing the case of the names
+
+import glob
+import os
+from os import path
+import re
+import shutil
+
+DST = '/tmp/placeholder_emoji_plus'
```
Comment (roozbehp) [inline build_emoji_set.py]: This should perhaps be a command-line argument.

**Comment by dougfelt** [comment]
@Behdad, can you take a look at the Makefile changes?

**Comment by dougfelt** [comment]
@behdad, can you comment on these changes?  are they ok?

**Comment by behdad** [comment]
Look great.  Thanks!

**Pull request merged**